src/playground/index.jsx

```diff
@@ -26,7 +27,39 @@ appTarget.className = styles.app;
 document.body.appendChild(appTarget);
 
 GUI.setAppElement(appTarget);
-const WrappedGui = HashParserHOC(AppStateHOC(GUI));
+
+// simple example of how you might manage project title externally.
+// Changing project title within GUI interface will update it here.
+class TitledGUI extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleUpdateProjectTitle'
+        ]);
+        this.state = {
+            projectTitle: 'Untitled-1'
```

Comment:
Would you please update this default to "Scratch Project"?  This came up in a planning meeting, and is only relevant here, in the standalone playground, not scratch-www.

Also, this string should use `formatMessage` so it can be translated.